webpack/scenes/ContentViews/Details/Versions/Delete/RemoveSteps/CVVersionDeleteFinish.js

```diff
@@ -15,50 +15,27 @@ const CVVersionDeleteFinish = () => {
     setIsOpen, selectedEnvSet,
     selectedCVForAK, selectedEnvForAK, selectedCVForHosts,
     selectedEnvForHost, affectedActivationKeys, affectedHosts,
-    deleteFlow, removeDeletionFlow, detailsPage,
+    deleteFlow, removeDeletionFlow,
   } = useContext(DeleteContext);
-  const removeCVVersionResponse = useSelector(state =>
-    selectRemoveCVVersionResponse(state, versionIdToRemove, versionEnvironments));
+
+  const dispatch = useDispatch();
   const removeCVVersionStatus = useSelector(state =>
     selectRemoveCVVersionStatus(state, versionIdToRemove, versionEnvironments));
-  const removeResolved = removeCVVersionStatus === STATUS.RESOLVED;
-  const dispatch = useDispatch();
   const [removeDispatched, setRemoveDispatched] = useState(false);
-  const [redirect, setRedirect] = useState(false);
   const selectedEnv = versionEnvironments.filter(env => selectedEnvSet.has(env.id));
+  const { push, location: { pathname } } = useHistory();
 
-  useDeepCompareEffect(() => {
-    if (removeResolved && removeCVVersionResponse && removeDispatched) {
-      dispatch(getContentViewVersions(cvId));
-      if (detailsPage) {
-        setRedirect(true);
-      } else {
-        setIsOpen(false);
-      }
+  useEffect(() => {
+    if (STATUS.ERROR) {
+      setIsOpen(false);
```

Comment:
Is this doing a if (removeCVVersionStatus === STATUS.ERROR) check? We can have that specified. This feels like magic.. :D

Reply:
lol.. I think that was a typo.